src/config/environment.rs

```diff
@@ -0,0 +1,51 @@
+use std::{
+    fmt::{Display, Formatter},
+    str::FromStr,
+};
+
+/// All the possible environments in which the application can run.
+#[derive(Eq, PartialEq, Debug, Clone, Copy)]
+pub enum Environment {
+    Dev,
+    Qa,
```

Comment:
Do we still have Qa?

Reply:
We plan to get rid of it but for now it's still a thing